gammapy/modeling/fit.py

```diff
@@ -767,37 +765,13 @@ def covariance_result(self):
         """Optimize result."""
         return self._covariance_result
 
-    def to_dict(self, full_output=False, overwrite_templates=False):
-        """Convert to dictionary."""
-
-        models_dict = self.models.to_dict(
-            full_output=full_output, overwrite_templates=overwrite_templates
-        )
-        output = {}
-        if self.optimize_result is not None:
-            output["optimize_result"] = self.optimize_result.to_dict()
-        if self.covariance_result is not None:
-            output["covariance_result"] = self.covariance_result.to_dict()
-        output["models"] = models_dict
-        return output
-
-    def to_yaml(self, full_output=False, overwrite_templates=False):
-        """Convert to YAML string."""
-        data = self.to_dict(
-            full_output=full_output, overwrite_templates=overwrite_templates
-        )
-        text = yaml.dump(
-            data, sort_keys=False, indent=4, width=80, default_flow_style=False
-        )
-        creation = CreatorMetaData()
-        return text + creation.to_yaml()
-
     def write(
         self,
         path,
         overwrite=False,
-        full_output=False,
+        full_output=True,
```

Comment:
I added this to have also the min/max on the parameters even if they are default because the FitResult should give the full information about the fit conditions.